model-optimizer/extensions/middle/PreserveRuntimeInfo.py

```diff
@@ -44,11 +45,37 @@ def preserve_rt_info(graph: Graph):
         for op in graph.get_op_nodes():
```

Comment:
Now it becomes long for loop. I propose to split this into two for loops so to have separate for `Parameter` and for `Result`